src/main/java/de/dhbw/mh/slang/codegen/JavaBytecodeGenerator.java

```diff
@@ -22,9 +22,9 @@ public JavaBytecodeGenerator( ){
 	@Override
 	public String visit( AstLiteral literal ){
 		if( literal.VALUE instanceof I8 ){
-			throw new RuntimeException( "not yet implemented" );
+			return String.format("sipush %s%n", ((I8) literal.VALUE).VALUE);
```

Comment:
I don't see where the test requires this line of code, but even if it did, it should say `bipush`.

Reply:
You missed this comment, please catch up.